app/assets/v2/js/pages/onboard.js

```diff
@@ -11,6 +11,15 @@ $('.js-select2').each(function() {
   $(this).select2();
 });
 
+// removes tooltip
+$('#job select').each(function(evt) {
+  $('.select2-selection__rendered').removeAttr('title');
+});
+// removes search field in all but the 'denomination' dropdown
+$('.select2-container').on('click', function() {
+  $('.select2-container .select2-search__field').remove();
```

Comment:
there is a built in option for select2 avoid showing the search input https://select2.org/searching#single-select